plugins/woocommerce-admin/client/homescreen/mobile-app-modal/pages/JetpackAlreadyInstalledPage.tsx

```diff
@@ -58,12 +59,13 @@ export const JetpackAlreadyInstalledPage: React.FC<
 	return (
 		<ModalContentLayoutWithTitle>
 			<>
+				<MobileAppInstallationInfo />
```

Comment:
Hmm the screenshot seems to not have been uploaded, could you try uploading this again? 🤔

Reply:
That's so weird that it linked to this PR.. 😵 

There's a scrollbar, indicating that the content has overflowed

![image](https://github.com/woocommerce/woocommerce/assets/27843274/f18da7f1-a8b2-42e4-b547-9d385da976d3)

Reply:
Good point! I updated the styles in https://github.com/woocommerce/woocommerce/pull/40559/commits/6a3a94ee42c5125ed4c7c4046f0ca8a2e898f2f2 so that this specific modal height fits to its content with a max height of 663px from my testing. Any other suggestions/improvements are welcome!

magic link available | magic link unavailable
-- | --
<img width="751" alt="Screenshot 2023-10-06 at 1 34 43 PM" src="https://github.com/woocommerce/woocommerce/assets/1945542/500f543d-a09d-45a5-be60-b2734f85c93b"> | <img width="760" alt="Screenshot 2023-10-06 at 1 35 22 PM" src="https://github.com/woocommerce/woocommerce/assets/1945542/a78f969b-0e19-4d22-8d7d-4e975eb46ff2">